app/scaffold-controllers/navigationController.js

```diff
@@ -149,5 +192,8 @@ function(
         return this.isActive;
     };
 
+    NavigationController.prototype.setActive = function(state) {
```

Comment:
Err sorry I should have mentioned this before. So there is an api standard (for plugins at least) that if a function takes in one boolean parameter then we split it into two functions (ex. show/hide methods for plugins). I would recommend doing the same thing so that the controller apis follow the same pattern as the plugin apis. So maybe `activate` and `deactivate` methods?